README.md

```diff
@@ -1,2 +1,114 @@
-# class-data-inheritable
-Class::Data::Inheritable - Inheritable, overridable class data
+NAME
```

Comment:
This, and all the other title sections, might be better looking as actual headings.

Reply:
I agree. I'm not sure how this would look on MetaCPAN, but I'll try and figure it out before next release.